packages/utils/test-tools/src/mockTheme.ts

```diff
@@ -0,0 +1,189 @@
+import { Theme, ColorRamp } from '@fluentui-react-native/theme-types';
```

Comment:
Presumably, this only copies what was being used in the tests? No additions or subtractions?

Reply:
Yep